backends/aptcc/apt-intf.cpp

```diff
@@ -2179,6 +2180,9 @@ bool AptIntf::runTransaction(const PkgList &install, const PkgList &remove, cons
             }
 
             m_cache->tryToRemove(Fix, verIt);
+
+            if (autoremove)
+                getDepends (removed_deps, verIt, true);
```

Comment:
`getDepends()` only considers Depends, not Pre-Depends, Recommends, or Suggests, and is thus incomplete.